libproto/src/router.rs

```diff
@@ -89,6 +89,7 @@ pub enum MsgType {
     RequestNewTxBatch,
     RequestNet,
     LocalSync,
+    RequestRpc,
```

Comment:
Why JSON-RPC call itself has to send a message via channel?

Reply:
At the first place can handle both http and websocket calls, the message has been convert to a mq message. If isn't sent via channel, it should be sent to mq and receive from mq.

Reply:
> Why JSON-RPC call itself has to send a message via channel?

Do you think it's better to send it to mq and receive it?